x-pack/legacy/plugins/ml/server/new_platform/plugin.ts

```diff
@@ -101,8 +114,18 @@ export interface UsageInitialization {
   savedObjects: SavedObjectsLegacyService;
 }
 
+declare module 'kibana/server' {
+  interface RequestHandlerContext {
+    ml?: {
+      mlClient: IScopedClusterClient;
+    };
+  }
+}
+
+export const PLUGIN_ID = 'ml';
```

Comment:
nit, it's good practice to keep const vars like this are the top of the file

Reply:
👌 Moved to top in 8c93d6b4a20f1de2821b6ab6f8835703bdb09abf